dask/dot.py

```diff
@@ -306,3 +308,237 @@ def graphviz_to_file(g, filename, format):
         f.write(data)
 
     return display_cls(filename=full_filename)
+
+
+def to_cytoscape_json(
```

Comment:
Given `dot` is the CLI for GraphViz, wonder if we should put this in a different module for organizational clarity

Reply:
Yeah, I waffled on whether to create a new module for this, but decided to keep them in the same module since they share some of the naming utilities.

What about changing this to `dask.visualize.py` (with perhaps a `dot.py` backwards compatibility shim)?